bin/build-ubuntu.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+# This script buildd BlazingMQ and all of its dependencies.
+#
+# Install the prerequisites, if not present yet:
+<<PREREQUISITES
+sudo apt update && sudo apt -y install ca-certificates
+sudo apt install -y --no-install-recommends \
```

Comment:
```suggestion
sudo apt-get update && sudo apt -y install ca-certificates
sudo apt-get install -y --no-install-recommends \
```
apt is designed for interactive use, this is a script so apt-get is more appropriate

Reply:
Well...thinking about it, the commands _are_ for interactive use. They are supposed to be copy-pasted by user.